components/subscription-base-infra/variables.tf

```diff
@@ -18,14 +18,6 @@ variable "location" {
   default     = "UK South"
 }
 
-variable "jenkins-ptlsbox-mi" {
+variable "jenkins_identity_object_id" {
```

Comment:
this should default to the prod object id, and remove from prod.tfvars, there's approx 6 envs this is used for while sbox is just sbox